projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-component.scss

```diff
@@ -558,6 +566,11 @@
         @extend %igx-grid__hierarchical-expander !optional;
     }
 
+    @include e(hierarchical-expander, $m: empty) {
+        @extend %igx-grid__hierarchical-expander !optional;
+        @extend %igx-grid__hierarchical-expander--empty !optional;
```

Comment:
I can't find any declaration for this placeholder selector? What's the purpose of this?

Reply:
The purpose is disable any pointer events and return the cursor to the default appearance for the expander cell in the pinned rows which doesn't have expander icon.

Reply:
It doesn't change the fact that there is no declaration of `%igx-grid__hierarchical-expander--empty` anywhere in the codebase. Hence, this does nothing.

Reply:
Here it is, I can use the full name if we have some conventions I missed:
https://github.com/IgniteUI/igniteui-angular/pull/6979/files#diff-e4bf11c573bc2e6e6f2423c2a1e70cb4R35